src/Core/Search/SearchParameters.php

```diff
@@ -58,10 +50,13 @@ public function __construct(AdminFilterRepository $adminFilterRepository)
     public function getFiltersFromRequest(Request $request, $filterClass)
     {
         $filters = [];
-        $defaultValues = $filterClass::getDefaults();
 
-        foreach (self::FILTER_TYPES as $type) {
-            $filters[$type] = $request->get($type, $defaultValues[$type]);
+        foreach (SearchParametersInterface::FILTER_TYPES as $type) {
```

Comment:
As this class implements the interface, isn't `self::FILTER_TYPES` still working?

Reply:
good question! I will try, that would be nice :P

Reply:
it works! thanks